src/web/components/MostViewed/MostViewedFooter/MostViewedFooter.tsx

```diff
@@ -81,95 +78,38 @@ const mostPopularAdStyle = css`
     ${labelStyles};
 `;
 
-const secondTierStyles = css`
-    border-left: 1px solid ${border.secondary};
-    border-right: 1px solid ${border.secondary};
-
-    ${from.tablet} {
-        padding-top: 24px;
-    }
-`;
-
 interface Props {
     sectionName?: string;
     pillar: Pillar;
     ajaxUrl: string;
 }
 
-function buildSectionUrl(ajaxUrl: string, sectionName?: string) {
-    const sectionsWithoutPopular = ['info', 'global'];
-    const hasSection =
-        sectionName && !sectionsWithoutPopular.includes(sectionName);
-    const endpoint: string = `/most-read${
-        hasSection ? `/${sectionName}` : ''
-    }.json`;
-    return joinUrl([ajaxUrl, `${endpoint}?dcr=true`]);
-}
-
-export const MostViewedFooter = ({ sectionName, pillar, ajaxUrl }: Props) => {
-    const url = buildSectionUrl(ajaxUrl, sectionName);
-    const { data, error } = useApi<
-        MostViewedFooterPayloadType | TrailTabType[]
-    >(url);
-
-    if (error) {
-        window.guardian.modules.sentry.reportError(error, 'most-viewed-footer');
-        return null;
-    }
-
-    if (data) {
-        return (
-            <div className={`content-footer ${cx(adSlotUnspecifiedWidth)}`}>
+export const MostViewedFooter = ({ sectionName, pillar, ajaxUrl }: Props) => (
+    <div className={`content-footer ${cx(adSlotUnspecifiedWidth)}`}>
+        <div
+            className={cx(stackBelow('leftCol'), mostPopularAdStyle)}
+            data-link-name="most-popular"
```

Comment:
do we need both ??
```
        data-link-name="most-popular"
        data-component="most-popular"
```

Reply:
I think these are used by the tracking scripts